tensorflow/core/kernels/control_flow_ops.cc

```diff
@@ -104,58 +98,54 @@ TF_CALL_ALL_TYPES(REGISTER_CPU_REF_SWITCH);
 TF_CALL_QUANTIZED_TYPES(REGISTER_CPU_SWITCH);
 TF_CALL_QUANTIZED_TYPES(REGISTER_CPU_REF_SWITCH);
 
-TF_CALL_NUMBER_TYPES_NO_INT32(REGISTER_GPU_SWITCH);
-TF_CALL_QUANTIZED_TYPES(REGISTER_GPU_SWITCH);
-TF_CALL_NUMBER_TYPES_NO_INT32(REGISTER_GPU_REF_SWITCH);
-TF_CALL_QUANTIZED_TYPES(REGISTER_GPU_REF_SWITCH);
-TF_CALL_variant(REGISTER_GPU_SWITCH);
-TF_CALL_bool(REGISTER_GPU_SWITCH);
-TF_CALL_bool(REGISTER_GPU_REF_SWITCH);
+TF_CALL_NUMBER_TYPES_NO_INT32(REGISTER_DEFAULT_SWITCH);
+TF_CALL_QUANTIZED_TYPES(REGISTER_DEFAULT_SWITCH);
+TF_CALL_NUMBER_TYPES_NO_INT32(REGISTER_DEFAULT_REF_SWITCH);
+TF_CALL_QUANTIZED_TYPES(REGISTER_DEFAULT_REF_SWITCH);
+TF_CALL_variant(REGISTER_DEFAULT_SWITCH);
+TF_CALL_bool(REGISTER_DEFAULT_SWITCH);
+TF_CALL_bool(REGISTER_DEFAULT_REF_SWITCH);
 
 #undef REGISTER_CPU_SWITCH
 #undef REGISTER_CPU_REF_SWITCH
-#undef REGISTER_GPU_SWITCH
-#undef REGISTER_GPU_REF_SWITCH
+#undef REGISTER_DEFAULT_SWITCH
+#undef REGISTER_DEFAULT_REF_SWITCH
 
-// Special GPU kernels for int32, string & resource handles. Requiring all
-// inputs and outputs to be in host memory.
-// TODO(b/25387198): Also enable int32 in device memory.
-#define REGISTER_GPU_HOST_KERNEL(type)                    \
+#define REGISTER_SPECIAL_DEFAULT_KERNEL(type)             \
```

Comment:
Let's remove `SPECIAL` from the name to keep it consistent with other naming.
```suggestion
#define REGISTER_DEVICE_DEFAULT_KERNEL(type)             \
```